README.md

```diff
@@ -81,9 +81,30 @@ After you have a **GraphServiceClient** that is authenticated, you can begin mak
 To retrieve the user's drive:
 
 ```Golang
+import (
+    "github.com/microsoftgraph/msgraph-sdk-go/models/odataerrors"
+)
+
+func printOdataError(err error) {
```

Comment:
could you move the function declaration below the call snippet please? I know this might not compile in real life but it's just for the sample clarity: people car about making the call first, handling potential errors second

Reply:
done